dizoo/d4rl/entry/d4rl_bcq_main.py

```diff
@@ -0,0 +1,21 @@
+from ding.entry import serial_pipeline_offline
+from ding.config import read_config
+from pathlib import Path
+
+
+def train(args):
+    # launch from anywhere
+    config = Path(__file__).absolute().parent.parent / 'config' / args.config 
```

Comment:
import from dizoo config file directly?

Reply:
yes, I copy this from other d4rl_*_main.py